decidim-templates/app/controllers/decidim/templates/admin/proposal_answer_templates_controller.rb

```diff
@@ -141,7 +141,7 @@ def availability_options_for_select
 
         def avaliablity_options
           @avaliablity_options = {}
-          Decidim::Component.includes(:participatory_space).where(manifest_name: :proposals)
+          Decidim::Component.includes(:participatory_space).where(manifest_name: accepted_components)
```

Comment:
I am not sure we need `accepted_components` here since this controller is explicitly for Proposal Answers. 
To be honest i have left it as raw as possible, for pending refactor.

Reply:
Well, it's because it facilitates overrides. We have a module that needs to use this too like proposals